source/blog/2020-10-27-release-notes.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: October 2020 Release Notes - Version 20.10.0 🎃
+published: true
+post_author:
+  display_name: Jing Yan
+  twitter: jyan
+tags: Releases, Finagle, Finatra, Scrooge, TwitterServer, Util
+---
+
+Trick or... our October release is as sweet as candies 👻
+
+### [Finagle](https://github.com/twitter/finagle/) ###
+
+#### Breaking API Changes
+
+-   finagle-thrift: Change the partition locator function getLogicalPartitionId in
+    PartitioningStrategy from Int =&gt; Int to Int =&gt; Seq\[Int\], which supports many to many mapping
```

Comment:
No idea whether the rendering of this line is correct, but just a chance to double check.

Reply:
good catch, updated!